src/components/empty_prompt/empty_prompt.stories.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiEmptyPrompt, EuiEmptyPromptProps } from './empty_prompt';
+
+import { EuiButton, EuiButtonEmpty } from '../button';
+import { EuiTitle } from '../title';
+import { EuiLink } from '../link';
+import { EuiImage } from '../image';
+import { EuiPageTemplate } from '../page_template';
+import illustration from '../../../src-docs/src/images/empty-prompt/illustration.svg';
+
+const meta: Meta<EuiEmptyPromptProps> = {
+  title: 'EuiEmptyPrompt',
+  component: EuiEmptyPrompt,
+  parameters: {
+    controls: {
+      exclude: ['data-test-subj'],
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<EuiEmptyPromptProps>;
+
+export const Playground: Story = {
+  args: {
+    title: <h2>Start adding cases</h2>,
+    layout: 'horizontal',
```

Comment:
let's default the playground to a color of `plain` to help distinguish it from the body background
```suggestion
    layout: 'horizontal',
    color: 'plain',
```

Reply:
Also it looks like the prop default for `layout` is `vertical`, not `horizontal`. Where it makes sense, let's follow the component default

```suggestion
    layout: 'vertical',
    color: 'plain',
```

In this case the transparent color default makes sense to deviate from because it makes the component easier to see and test.